contracts/WildcardSteward_v2.sol

```diff
@@ -177,14 +191,25 @@ contract WildcardSteward_v2 is Initializable {
             assetToken.mintWithTokenURI(address(this), tokens[i], tokenUri);
             benefactors[tokens[i]] = _benefactors[i];
             state[tokens[i]] = StewardState.Foreclosed;
-            artistPercentages[tokens[i]] = 100;
             timeLastCollected[tokens[i]] = now;
             patronageNumerator[tokens[i]] = _patronageNumerator[i];
             tokenGenerationRate[tokens[i]] = _tokenGenerationRate[i];
+            if (_releaseDate[i] == 0) {
```

Comment:
```suggestion
            if (_releaseDate[i] < now) {
```

Reply:
To catch any mistakes (we don't want to give tokens away for the minimum price without having the auction first.